src/main/java/com/gtnewhorizons/modularui/common/widget/textfield/BaseTextFieldWidget.java

```diff
@@ -359,6 +367,22 @@ public BaseTextFieldWidget setFocusOnGuiOpen(boolean focused) {
         return this;
     }
 
+    public BaseTextFieldWidget setMaximumFractionDigits(int digits) {
+        if (decimalFormat != null) {
```

Comment:
then this null check is effectively unneeded

Reply:
unresolved

Reply:
My bad